src/Mod/Draft/draftmake/make_sketch.py

```diff
@@ -38,11 +38,11 @@
 from draftutils.translate import translate
 
 
-def make_sketch(objectslist, autoconstraints=False, addTo=None,
-                delete=False, name="Sketch", radiusPrecision=-1):
-    """makeSketch(objectslist,[autoconstraints],[addTo],[delete],[name],[radiusPrecision])
+def make_sketch(objectsList, autoconstraints=False, addTo=None,
```

Comment:
`snake_case`. I know many arguments are `camelCase`, but this is an old style, not compliant with our modern coding conventions.